README.md

```diff
@@ -26,11 +26,11 @@ If you haven't used Twilio before, welcome! You'll need to [Sign up for a Twilio
 
 ### CocoaPods
 
-1. Install [CocoaPods 1.0.0+](https://guides.cocoapods.org/using/getting-started.html).
+1. Install [CocoaPods 1.7.5 or newer](https://guides.cocoapods.org/using/getting-started.html).
```

Comment:
We don't have a "common issues" section, but I guess if we did I would mention that duplicate category definitions can be solved by upgrading CocoaPods.

Reply:
I guess it wouldn't hurt to add that section?

Reply:
I looked and wasn't really sure where to add it. I think the QS README needs a bit of an overhaul to be closer to what the Voice team is doing.